### PR TITLE
ss-41: Add dashboard page scaffold

### PR DESCRIPTION
**Pull request by Holovashcenko**
Add Dasboard
Add styles
Add structure
image

### CODE REVIEW

**Review by VItaliy-Kalenichenko — CHANGES_REQUESTED** [review]
Create a route /dashboard for this page, so we can test it

**Comment by Holovashcenko** [comment]
There is already exist a route on app page but it's private.
I added it to ui page

**Review by nikita-remeslov — COMMENTED** [review]
Looks good, please fix conflicts

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Button, Input } from '../../components/components';
+import { DoughnutChart } from '../../components/doughnut-chart/doughnut-chart';
```
Comment (teslmik) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:5]: Fix import, should be from component.ts file

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                }}
+            >
+                {children}
+            </div>
+        </div>
+    );
+};
```
Comment (teslmik) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:79]: Can it be moved to a separate component?

File: frontend/src/bundles/common/pages/dashboard/dashboard.tsx (hunk 1 above)
Comment (teslmik) [inline frontend/src/bundles/common/pages/dashboard/dashboard.tsx:5]: Fix import, should be from component.ts file
`import { Button, Input, DoughnutChart } from '../../components/components';`
My mistake, didn't add the Doughnut chart to the common components file

Comment (Holovashcenko) [reply frontend/src/bundles/common/pages/dashboard/dashboard.tsx]: Fixed in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/65/commits/36179e20e82c40c26e89ba2ef454742920f5dc18](url)

**Comment by nikita-remeslov** [comment]
> Create a route /dashboard for this page, so we can test it

@Holovashcenko Not resolved.

**Comment by Holovashcenko** [comment]
This route has already existed, but it is a private route.

**Comment by Holovashcenko** [comment]
Remove dashboard from UI page. Dashboard will be able to test when user is authenticated.
image

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**